std_msgs/Bool rob_a
std_msgs/Bool rob_b
std_msgs/Bool rob_c
std_msgs/Bool rob_d
